internal/indexer/indexer.go

```diff
@@ -574,11 +588,37 @@ func (i *Indexer) ensureRangeFor(pos token.Position, obj types.Object) uint64 {
 		return rangeID
 	}
 
-	rangeID = i.emitter.EmitRange(start, end)
+	rangeID = i.emitter.EmitRange(start, end, nil)
 	i.ranges[pos.Filename][pos.Offset] = rangeID
 	return rangeID
 }
 
+func (i *Indexer) emitRangeForSymbol(pos token.Position, length int, tag *protocol.RangeTag) uint64 {
+	i.stripedMutex.LockKey(pos.Filename)
+	defer i.stripedMutex.UnlockKey(pos.Filename)
+
+	line := pos.Line - 1
+	column := pos.Column - 1
+
+	start := protocol.Pos{Line: line, Character: column}
+	end := protocol.Pos{Line: line, Character: column + length}
+
+	if rangeID, ok := i.ranges[pos.Filename][pos.Offset]; ok {
+		log.Printf("Duplicate range already exists at %s:%d, tag may be absent", pos.Filename, pos.Offset)
```

Comment:
Can you guard this output like we do with the rest? There's verbosity levels we can use here (and there's also output animation that rogue prints will mess up).